clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesPodHandler.java

```diff
@@ -60,7 +63,34 @@ public SpinnakerKind spinnakerKind() {
 
   @Override
   public Status status(KubernetesManifest manifest) {
-    return new Status();
+    Status result = new Status();
+    V1Pod pod = KubernetesCacheDataConverter.getResource(manifest, V1Pod.class);
+    V1PodStatus status = pod.getStatus();
+
+    if (status == null) {
+      result.unstable("No status reported yet")
+          .unavailable("No availability reported");
+      return result;
+    }
+
+    // https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/
+    String phase = status.getPhase();
+
+    if (phase == null ) {
+      result.unstable("No phase reported yet")
+          .unavailable("No availability reported");
+    } else if (phase.equals("pending")) {
+      result.unstable("Pod is 'pending'")
+          .unavailable("Pod is still starting");
```

Comment:
"Pod is waiting to start" is how I'd put it. To me, `ContainerCreating` is the "still starting" phase, while `Pending` is "we haven't decided where to try starting this", but maybe that's just me.

Reply:
No you're right, will change